### ISSUE TITLE
[Question] Are you accepting contributions?

### ISSUE THREAD

**Issue by mr-adult** [hargata/lubelog#109]
I've been impressed by this little app but notice some visual hickups in various places that I would like to fix. Are you accepting contributions? If so, is there a preferred way you would like contributions? For example, via fork and pull request or some other mechanism?

**Comment by hargata** [comment]
I am yeah, but biggest problem IMO is that this is under super active development and so there's a good chance a lot of the visual bugs are already being fixed as we speak. First step is to probably list out the visual hickups and they'll likely be fixed by next release.

**Comment by mr-adult** [comment]
Fair enough. I'll just write them up for now and start contributing once the development cools off a bit.
